### PR TITLE
fix: replace base64 spinners with circular progress component

### PR DESCRIPTION
**Pull request by Shijir**
Signed-off-by: stsogoo <stsogoo@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by Shijir** [comment]
Before (Left) and After (Right) screenshots 
Screen Shot 2020-09-29 at 10 39 19 AM
Screen Shot 2020-09-29 at 10 38 36 AM
Screen Shot 2020-09-29 at 10 36 39 AM

File: packages/core/src/alert/alert.element.scss [hunk 1]
```diff
@@ -1,9 +1,6 @@
 @import './../styles/tokens/generated/index';
 @import './../styles/mixins/mixins';
 
-// TODO: progress spinner CSS deprecated, will be replaced by cds-circular-progress
-@import './../styles/utils/legacy-spinner';
```
Comment (mathisscott) [inline packages/core/src/alert/alert.element.scss:5]: 🎉

File: packages/core/src/button/base-button.element.scss [hunk 2]
```diff
@@ -70,6 +67,10 @@
     cds-icon[shape='error-standard'] {
       --color: #{$cds-token-color-danger-700};
     }
+
+    .button-spinner {
+      --ring-color: var(--cds-token-color-neutral-400);
```
Comment (mathisscott) [inline packages/core/src/button/base-button.element.scss:72]: 👍

**Review by mathisscott — APPROVED** [review]

**Comment by mathisscott** [comment]
Looks like we have some unit tests failing...

**Review by jeeyun — APPROVED** [review]

**Pull request merged**